### ISSUE TITLE
OASYS - Info block - Sys Plot (Top View)

### ISSUE THREAD

**Issue by Telschazz** [oasys-kit/ShadowOui#234]
I had problems understanding the plot in "Sys Plot (Top View)" of "Shadow PostProcessor -> Info".
In my mind the term "top view" implies a vantage point above the beamline. Unfortunately the X coordinate in the plot is displayed upward by default which would be a bottom view in Shadow coordinates. So if you have a "Plane Mirror" with "O.E. Orientation angle [deg]" of 90 degrees, the "Top Plot" seems to indicate a deflection to the right:
![grafik](https://user-images.githubusercontent.com/11213750/188090396-acdb2996-6829-43d0-891e-965d6e9c7bef.png)

Could the plot be displayed with X coordinate downward by default instead to make it easier for newbies like me?

**Comment by srio** [comment]
Hello, 
I do not understand what is the problem. You clearly see that, when you travel with the beam, the deflection is to the right, as you said. The view is from a point at z positive and large (z>>0). Note that here the x,y,z are the LAB coordinates, not the screen or mirror coordinates.
Rgds, M

**Comment by srio** [comment]
Yes, I see. The top view in the LAB system ( (a) in the picture) would correspond to positive x's looking down. 
I kept this issue open until that is changed. 
Thanks a lot for reporting this.

**Comment by lucarebuffi** [comment]
I warmly suggest to avoid this sarcastic and disrespectful tone. We are trying to help, as we do with all the users. Please reconsider the approach to this interaction with us, otherwise I will close the issue and you'll find your answers elsewhere.

Best, L.R.

**Comment by Telschazz** [comment]
I meant no offense!
As I said I'm a newbie and not sure whether this has any point or what I'm missing to see the picture.
Please tell me what offended you and I'll edit my comment.

**Comment by Telschazz** [comment]
I think everything is plotted correct. 
If you know the orientation of the lab system, where -X is left and +X is right, everything is fine.
But the wording of "Top view" with the X[mm]-axis oriented upward seems to imply otherwise.

**Comment by lucarebuffi** [comment]
Thanks for clarifying. Hopefully we will be able to fix the issue asap.
